scripts/installs/start_glassfish_service.bat

```diff
@@ -1 +1,3 @@
-net start "domain1 GlassFish Server"
\ No newline at end of file
+Taskkill /IM domain1Service.exe /F
+ping 192.168.200.201 -n 1 -w 1000 > nul
```

Comment:
Ok. I find some commands that can works but can depend of windows version:
- `timeout /t 1` available since win7 (https://technet.microsoft.com/en-us/library/cc754891.aspx)
- `powershell -command "Start-Sleep -s 1"` requires powershell

Source: http://stackoverflow.com/questions/1672338/how-to-sleep-for-5-seconds-in-windowss-command-prompt-or-dos
(They also mention the `ping` hack)